projects/ngneat/reactive-forms/src/lib/formGroup.ts

```diff
@@ -269,4 +272,18 @@ export class FormGroup<T extends Obj = any, E extends object = any> extends NgFo
   setDisable(disable = true, opts?: ControlEventOptions) {
     disableControl(this, disable, opts);
   }
+
+  persist(
+    key: string,
+    debounceTime: number = 250,
+    manager: PersistManager<T> = new LocalStorageManager<T>()
+  ): Observable<T> {
+    const formValue = manager.getValue(key);
+    this.reset(formValue);
```

Comment:
How should we deal with FormArrays here? Can't really make any assumptions on how the controls inside the form arrays should be created (e.g. validators etc.)

Reply:
https://github.com/ngneat/forms-manager#using-formarray-controls

Reply:
Ah, I had something like this in mind, I'll check it out in the evening.

Reply:
Ok, added the FormArray factory options. Also managed to limit the config to arrays and correctly infer the value types for the factory functions.